### PR TITLE
Different gradle tasks for integration testing depending on the DBMS

### PR DESCRIPTION
**Pull request by marijnritense**

### CODE REVIEW

File: gradle/test.gradle [hunk 1]
```diff
@@ -26,8 +26,9 @@ test {
     }
 }
 
-tasks.register("integrationTesting", Test) {
+tasks.register("integrationTestingPostgresql", Test) {
```
Comment (theo-ritense) [inline gradle/test.gradle:33]: Please add an alias for backwards compatibility with the pipeline and ease of use.
```suggestion
tasks.register("integrationTesting") {
    dependsOn("integrationTestingPostgresql")
}

tasks.register("integrationTestingPostgresql", Test) {
```

File: docker-resources/docker-compose-base-test-mysql.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            --sql-mode=NO_ENGINE_SUBSTITUTION
+        #ports:
+           # - "5432:5432" #needs overriding "XXXX:5432" XXXX=your-own-port-per-module-test
+        environment:
+            MYSQL_ROOT_PASSWORD: password
+            MYSQL_DATABASE: valtimo #needs overriding
+            MYSQL_USER: valtimo
+            MYSQL_PASSWORD: password
+# Ports taken
+#   url: jdbc:postgresql://localhost:3301/form-test
```
Comment (theo-ritense) [inline docker-resources/docker-compose-base-test-mysql.yml]: I would not expect jdbc:postgresql urls here. Maybe move the 'Ports taken' somewhere else, or only keep it at the postgres declaration.

File: objects-api/src/test/resources/config/document/application-mysql.yml
Comment (theo-ritense) [inline objects-api/src/test/resources/config/document/application-mysql.yml:1]: why do we need these configs in `/config/document/` ?

File: outbox/outbox-rabbitmq/src/test/kotlin/com/ritense/outbox/rabbitmq/RabbitMessagePublisherIntTest.kt [hunk 3]
```diff
@@ -54,7 +55,8 @@ class RabbitMessagePublisherIntTest {
     @Nested
     @ActiveProfiles("invalidrouting")
     inner class InvalidRouting @Autowired constructor(
-        val springCloudMessagePublisher: RabbitMessagePublisher
+        val springCloudMessagePublisher: RabbitMessagePublisher,
+        val environment: Environment
```
Comment (theo-ritense) [inline outbox/outbox-rabbitmq/src/test/kotlin/com/ritense/outbox/rabbitmq/RabbitMessagePublisherIntTest.kt]: I this a remnant of your attempt to fix some tests?

Comment (marijnritense) [reply outbox/outbox-rabbitmq/src/test/kotlin/com/ritense/outbox/rabbitmq/RabbitMessagePublisherIntTest.kt]: It is, thanks for pointing it out. I removed it.

File: objects-api/src/test/resources/config/document/application-mysql.yml
Comment (marijnritense) [reply objects-api/src/test/resources/config/document/application-mysql.yml:1]: We don't, I removed them

File: docker-resources/docker-compose-base-test-mysql.yml (hunk 2 above)
Comment (marijnritense) [reply docker-resources/docker-compose-base-test-mysql.yml]: I changed `postgresql` to `mysql` for now.

**Review by theo-ritense — APPROVED** [review]

**Pull request merged**